cmd/docgenerate/main.go

```diff
@@ -46,15 +47,19 @@ func main() {
 		panic(err)
 	}
 
-	// Generate guides and reference docs
-	mustRunIn(path.Join(srcdir, "docs"), "md2html", "build", outdir)
-
-	// Generate SDK-specific documentation
 	wg := new(sync.WaitGroup)
+
+	// Generate SDK-specific documentation and search index files
 	for _, v := range versionPaths(srcdir) {
 		wg.Add(1)
 		go makeSdkDocs(wg, v, srcdir, outdir)
+		wg.Add(1)
+		go makeIndexInputFiles(wg, v, srcdir)
 	}
+
+	// Generate guides and reference docs
```

Comment:
:+1: Ah ha, gotcha. Might be worth adding a comment to that effect. So this also quietly updates your local `searchIndex.js` files in `$CHAIN`, and they get copied out after the fact.